src/actions/save/index.js

```diff
@@ -51,10 +53,22 @@ export const createSaveAction: SaveAction = (config: SaveActionConfig) => {
 
   return {
     type: "save",
-    save: () => {
+    save: (onError, paymentSourceDetails) => {
+      const { createVaultSetupToken } = config;
+    
+      if (!onError || !paymentSourceDetails) {
+        throw new ValidationError("Missing args to #save")
```

Comment:
Should this return a rejected ZalgoPromise instead? (Assuming there is such a thing. Forgive my ZalgoPromise ignorance.)

Reply:
Honestly, uh, maybe? There are no docs on ZalgoPromise so I'm really not sure. But, I think you may be right, it should probably be an explicit reject instead.